app/main/forms.py

```diff
@@ -1248,36 +1245,36 @@ def validate_template_content(self, field):
         BroadcastLength()(None, field)
 
 
-class LetterAddressForm(StripWhitespaceForm):
-    def __init__(self, *args, allow_international_letters=False, **kwargs):
-        self.allow_international_letters = allow_international_letters
-        super().__init__(*args, **kwargs)
+# class LetterAddressForm(StripWhitespaceForm):
```

Comment:
Same for this file, shouldn't we just remove it? If we find we need it back we can pull it in from an older commit